com.unity.render-pipelines.core/Runtime/Debugging/DebugUI.Fields.cs

```diff
@@ -328,8 +345,7 @@ internal void InitQuickSeparators()
 
             internal void InitIndexes()
             {
-                if (enumNames == null)
-                    enumNames = new GUIContent[0];
+                enumNames ??= new GUIContent[0];
```

Comment:
Actually I had to switch to the beta branch of the formatter to get this in properly :(

Reply:
what happens if I launch the formatter and I am not in the beta branch?

Reply:
Hmm, in fact I see now Yamato formatting has failed because of this. I've opened a slack thread for advice on what to do:https://unity.slack.com/archives/CU41BKSJF/p1619512249177000

Reply:
I've reverted the use of ??=.